config/default.yml

```diff
@@ -547,6 +547,7 @@ Rails/Pluck:
   StyleGuide: 'https://rails.rubystyle.guide#pluck'
   Enabled: 'pending'
   VersionAdded: '2.7'
+  VersionChanged: '<<next>>'
```

Comment:
Can you remove it because the default configuration of this cop does not change.

Reply:
Sure, done.